### PR TITLE
REX last 11 cards 

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/cresting_mosasaurus.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Cresting Mosasaurus
+ManaCost:6 U U
+Types:Creature Dinosaur
+PT:4/8
+K:Emerge:6 U
+T:Mode$ ChangesZone | ValidCard$ Card.wasCastFromYourHandByYou+Self | Destination$ Battlefield | Execute$ TrigBounceAll | TriggerDescription$ When CARDNAME enters the battlefield, if you cast it, return each non-Dinosaur creature to its owner's hand.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/cresting_mosasaurus.txt:6]: wrong valid

File: forge-gui/res/cardsfolder/upcoming/spitting_dilophosaurus.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Name:Spitting Dilophosaurus
+ManaCost:2 B
+Types:Creature Dinosaur
+PT:3/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigPutCounter | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, put a -1/-1 counter on up to one target creature.
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPutCounter | Secondary$ True | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, put a -1/-1 counter on up to one target creature.
+SVar:TrigPutCounter:DB$ PutCounter | ValidTgts$ Creature | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target creature | CounterType$ M1M1 | IsCurse$ True | CounterNum$ 1
+S:Mode$ Continuous | Affected$ Creature.counters_GE1_M1M1+OppCtrl | AddHiddenKeyword$ CARDNAME can't attack or block. | Description$ Creatures your opponents control with -1/-1 counters on them can't block.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/spitting_dilophosaurus.txt:8]: wrong KW

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/spitting_dilophosaurus.txt:8]: better write as static

File: forge-gui/res/cardsfolder/upcoming/grim_giganotosaurus.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Name:Grim Giganotosaurus
+ManaCost:5 B G
+Types:Creature Dinosaur
+PT:10/10
+K:Monstrosity:10:10 B G:X
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/grim_giganotosaurus.txt:5]: @Northmoc 
can you fix the X description? 🙏

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/grim_giganotosaurus.txt:5]: image
I think I got it fixed myself

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/grim_giganotosaurus.txt]: You're still showing the fresh view that's only based on Oracle
Here's the parsed version:
image

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/grim_giganotosaurus.txt]: I think you fixed it ? 
image

**Review by tool4ever — APPROVED** [review]

**Pull request merged**